src/markdown-pages/build-apps/create-an-ab-test-application/catalog-info.mdx

```diff
@@ -38,13 +38,13 @@ cd nru-programmability-course/describe-app/ab-test
 
```

Comment:
Similar to the `Update your Nerdpack's catalog information` guide, how about we update the "big-picture" framing  up on line 25 to have plural catalog directories:

`To supply information to the catalog about your app, you need to create the _catalog_ directory in your Nerdpack.`

-> 

`To supply information to the catalog about your app, you need to create _catalog_ directories in your Nerdpack.`

Reply:
cc @alexronquillo

Reply:
> ...you need to create _catalog_ directories...

Are multiple _catalog_ directories actually required or are they just automatically created? In another comment there was a conversation about whether empty _screenshots_ directories are skipped or not. I think that may factor in here too. 

In general, I think the change is fine, except that this guide might need to be reworked a little bit more because the procedures following the create command only deal with adding info to the root-level _catalog_ directory.

If these Nerdpack item-level catalog directories can be deleted/skipped, then we may need to add a step to do that. Or maybe I'm overthinking this? 😅

Reply:
Multiple catalog directories are automatically created but not required. Totally optional if you want to add nerdpack item screenshots or not.

We will be releasing an update to the CLI that will skip uploading any screenshots directory that is empty. Currently it is a bit annoying. If you run the `nr1 create -t catalog` then try and submit, you'll get an error stating that you need at least 1 screenshot. That shouldn't be the case, so a patch is incoming.

Reply:
Ok, I think if the document describes the final state correctly but there's effectively a bug in the code, then let's just make the change to plural and be done with it 😃